site/topics/conditions/conditional-jump.rst

```diff
@@ -23,6 +23,61 @@ Including the Flag Register in the System
 Programming with Conditional Jumps
 ==================================
 
+* With this new functionality, conditional jumps, branching can now be achieved
+
+
+* Consider the problem discussed before
+
+    * Given some number, output ``1`` if it is less than ``10``, otherwise, output ``0``
+
+
+* This problem can be solved with the following general idea
+
+    * Load value into register A
+    * Subtract ``10`` from the value
+
+        * If the result is negative (the most significant bit is high), the value must be ``< 10``
+
+
+    * If the result is negative, jump to a part of RAM that outputs ``1``
+
+        * ``JMPS``
+
+
+    * If the result is not negative, output ``0``
+
+
+* Below is the ESAP system's machine code for the above idea
+
+    * The emphasized line contains the conditional jump for the significant bit/sign flag
+    * Here, the value in RAM address 15 is the number to check
+
+.. literalinclude:: check_10.hex
+    :language: text
+    :lineno-match:
+    :emphasize-lines: 5
+
+
+* Unfortunately, the machine code in hex format is hard to interpret
+* To make this more human readable, the program can be explained as follows
+
+    0. Load the value from address 15 to register A
+    1. Load the value 10 to register B
+    2. Calculate the difference
+    3. Jump to address 6 if the significant bit/sign flag is high
```

Comment:
Maybe a literal code block here is a better style. Are these actually numbered as `0,1,2,3...`, or is this rendered as `1,2,3,4...` ? Markdown ignores the numeric values typically (you'll see `1..1..1` for numbered lists typcally).

Reply:
Good point --- checked, and it was good. 

![image](https://github.com/user-attachments/assets/1a6638c8-3098-417f-be1e-2e266ffa79b2)

Nevertheless, I do like the idea of the literal block --- changing

Reply:
It loses the counting at 0, but whatever, the style is better this way. 

![image](https://github.com/user-attachments/assets/814d1e59-1905-4eae-84fd-32f53813518d)